src/mixins/people-roles-mixin.ts

```diff
@@ -86,6 +86,11 @@ export default class PeopleRolesMixin extends Vue {
     return this.orgPersonList.filter(person => this.isProprietor(person))
   }
 
+  /** The list of partners. */
+  get partners (): OrgPersonIF[] {
+    return this.orgPersonList.filter(person => this.isPartner(person))
+  }
```

Comment:
Prep for GP, although there is some question about whether to differentiate Partners from Proprietors. I think they're the same, except SPs can have only 1 and GPs need at least 2.

Anyway, this is Future Severin's problem :)